index.js

```diff
@@ -134,11 +139,17 @@ async function cmd_convert(argv) {
     }
   } else {
     res = await redashSQL(`
-      SELECT ${columns.join(",")}
-      FROM addons
+      SELECT ${columns.map(col => "a." + col).join(",")}
+      FROM addons a
+      ${argv.wx ? `
+        LEFT JOIN versions v ON (v.addon_id = a.id)
+        LEFT JOIN files f ON (f.version_id = v.id)
+        ` : ""}
       WHERE
-        ${argv.input} IN (${escapedIds.join(",")})
-        AND guid NOT LIKE 'guid-reused-by-pk-%'
+        a.${argv.input} IN (${escapedIds.join(",")})
+        AND a.guid NOT LIKE 'guid-reused-by-pk-%'
```

Comment:
It would be great if we could find a better strategy than just excluding re-used ones. They are still valid and subject of interest.

Reply:
Definitely something to consider, though not something I'm changing for this commit. I'm more worried that the joining and grouping could give us to too many or too few results.